src/client/pages/OfferNew/Checkout/index.tsx

```diff
@@ -372,14 +474,28 @@ export const Checkout = ({
                     email={offerData.person.email ?? ''}
                     onEmailChange={onEmailChange}
                     ssn={offerData.person.ssn ?? ''}
-                    ssnBackendError={ssnBackendError}
                     onSsnChange={onSsnChange}
+                    ssnBackendError={ssnBackendError}
                     isFirstAndLastNameVisible={
                       !offerData.person.firstName || !offerData.person.lastName
                     }
+                    onSubmit={startSign}
                   />
-                </CheckoutContent>
-
+                  <StartDateWrapper>
+                    <StartDateLabel>
+                      {textKeys.SIDEBAR_STARTDATE_CELL_LABEL()}
+                    </StartDateLabel>
+                    <StartDate offerData={offerData} refetch={refetch} />
+                  </StartDateWrapper>
+                  {locale.marketLabel == 'SE' &&
+                    renderUpsellCard(
+                      quoteBundleVariants,
+                      selectedQuoteBundleVariant,
+                      onAddQuotes,
+                      textKeys,
+                    )}
```

Comment:
See above -- I don't see why this shouldn't be a normal component => `<UpsellCard />`